doc/ansible/dev/testing_guide.md

```diff
@@ -4,15 +4,14 @@
 
 ### Requirements
 To begin, you sould have:
+
 - The latest version of the [operator-sdk](https://github.com/operator-framework/operator-sdk) installed.
 - Docker installed and running
-- [Molecule](https://github.com/ansible/molecule) >= 3.0
+- [Molecule](https://github.com/ansible/molecule) >= v3.0
 - [Ansible](https://github.com/ansible/ansible) >= v2.9
 - [The OpenShift Python client](https://github.com/openshift/openshift-restclient-python) >= v0.8
-- An initialized Ansible Operator project, with the molecule directory present. If you initialized a project with a previous
-  version of operator-sdk, you can generate a new dummy project and copy in the `molecule` directory. Just be sure
-  to generate the dummy project with the same `api-version` and `kind`, or some of the generated files will not work
-  without modification. Your top-level project structure should look like this:
+- An initialized Ansible Operator project, with the molecule directory present. If you initialized a project with a previous version of operator-sdk, you can generate a new dummy project and copy in the `molecule` directory. Just be sure to generate the dummy project with the same `api-version` and `kind`, or some of the generated files will not work without modification. Your top-level project structure should look like this:
+
```

Comment:
If you check the layout of this doc is broken, so all changes made was just in its layout.

Reply:
The first sentence makes sense in this format, but the rest of it should be moved into a note outside of the bulleted list.

Reply:
sorry, I could not catcher what do you mean with your comment. Just to clarifies the change performed here, See:

Before (layout with bullet points but broken )

<img width="910" alt="Screenshot 2020-04-13 at 21 45 41" src="https://user-images.githubusercontent.com/7708031/79159519-25dafe80-7dd0-11ea-8ff2-b68b417200a0.png">

After/Pr changes (same but fixed )

<img width="1022" alt="Screenshot 2020-04-13 at 21 45 02" src="https://user-images.githubusercontent.com/7708031/79159546-2ffcfd00-7dd0-11ea-9009-36ecf03b03b8.png">

so, could you please let me know what exactly do you think is wrong here and how should be?

Reply:
I see, the word wrapping seems to only matter in bulleted lists, not regular paragraphs though. 

Regarding the first sentence, "An initialized Ansible operator project with the molecule directory present". The rest of that block should be a note. The long paragraph is only necessary in a subset of cases, and shouldnt be bloating an otherwise concise list.

Reply:
Done. 👍